app/assets/stylesheets/home.scss

```diff
@@ -21,6 +21,10 @@ $white: #ffffff;
 .search {
   background-color: #FF851B;
   color: white;
+  &:hover {
+    background-color: #FF851B;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.